core/src/sampling/RandomSampler.cpp

```diff
@@ -142,7 +139,7 @@ void RandomSampler::draw(std::vector<size_t>& result,
   if (num_samples < max / 2) {
```

Comment:
I think this should now be changed to `max / 10`, following ranger's approach.

Reply:
Sorry for making you point this out twice.